app/controllers/provider/admin/authentication_providers_controller.rb

```diff
@@ -3,7 +3,7 @@
 class Provider::Admin::AuthenticationProvidersController < FrontendController
   before_action :authorize_settings
 
-  activate_menu :audience, :cms, :oauth2
+  activate_menu :audience, :cms, :sso_integrations
```

Comment:
@thomasmaas 
There are 2 authentication provider controllers: one calls `:sso_integrations` and this one `oauth2`.

In `vertical_nav_helper` both menus have same id `:sso_integration`. Do you know if this ID matters?

We could change it here or in `vertical_nav_helper`:
https://github.com/3scale/porta/blob/b376edcebf7bf56673201ab30c091031aaba5ac2/app/helpers/vertical_nav_helper.rb#L151

Reply:
There is no `:oauth2` in this helper in the activate_sidebar (third parameter is sidebar

https://github.com/3scale/porta/blob/master/app/lib/menu_system.rb#L62-L64

I think @srijitamukherjee is right

Reply:
> There is no :oauth2 in this helper in the activate_sidebar (third parameter is sidebar

Yep I'm aware, I did that. But I could have been wrong, I'm asking if `oauth2` is relevant for something else than the sidebar. Just in case. @thomasmaas wrote that line so maybe he knows.

Reply:
oauth2 has been deprecated

Reply:
and will not make it to portafly